### ISSUE TITLE
Symfony classes missing

### ISSUE THREAD

**Issue by alfechner** [netz98/n98-magerun#493]
I installed n98 with both stated methods. In both cases I get the following errors.

```
PHP Fatal error:  Class 'Symfony\Component\Console\Event\ConsoleTerminateEvent' not found in phar:///usr/local/bin/n98-magerun.phar/vendor/symfony/console/Symfony/Component/Console/Application.php on line 884

Fatal error: Class 'Symfony\Component\Console\Event\ConsoleTerminateEvent' not found in phar:///usr/local/bin/n98-magerun.phar/vendor/symfony/console/Symfony/Component/Console/Application.php on line 884
```

Any suggestions? Thanks!

**Comment by tkn98** [comment]
If I may ask you to highlight the differences to the existing issues carying the same error message (#221, #292 and ...) and tell how your report is different to those (please reference each one while you outline the differences).

**Comment by alfechner** [comment]
This issue is already solved here: #221. Sorry for opening a redundant one.

**Comment by tkn98** [comment]
No problem at all.
